server/common/message.go

```diff
@@ -131,6 +132,21 @@ func MakeStatusMessage(s int) []byte {
 	msg := DrawResponseMsg{
 		Type:   DrawResponse,
 		Status: s,
+		RemainingTime: -1,
```

Comment:
if we're going to use a `DrawResponse` message here can we always set the cooldown time remaining? looks like we only have to update in two locations:

```
gvacaliuc@packy:~/.../rgreen312/owlplace$ rg MakeStatusMessage
server/common/message.go
130:func MakeStatusMessage(s int) []byte {

server/wsutil/conn.go
101:		message := common.MakeStatusMessage(503)
109:		message := common.MakeStatusMessage(503)
```

however if we return a server error i suppose that the value of the cooldown doesn't really matter xD